inlang/source-code/cli/src/commands/lint/index.ts

```diff
@@ -3,11 +3,12 @@ import Table from "cli-table3"
 import { getInlangProject } from "../../utilities/getInlangProject.js"
 import { log } from "../../utilities/log.js"
 import type { InlangProject, MessageLintReport } from "@inlang/sdk"
+import { projectOption } from "../../utilities/globalFlags.js"
 
 export const lint = new Command()
 	.command("lint")
 	.description("Commands for linting translations.")
-	.requiredOption("--project <path>", "Path to the project settings file.", "./project.inlang.json")
+	.requiredOption(projectOption.flags, projectOption.description, projectOption.defaultValue)
```

Comment:
I think you can spread the project option

Reply:
no somehow this raised type issues thus typed array.